packages/alfa-rules/test/sia-r35/rule.spec.tsx

```diff
@@ -0,0 +1,137 @@
+import { h } from "@siteimprove/alfa-dom";
+import { None, Option } from "@siteimprove/alfa-option";
+import { test } from "@siteimprove/alfa-test";
+
+import R35, { Outcomes } from "../../src/sia-r35/rule";
+
+import { evaluate } from "../common/evaluate";
+import { oracle } from "../common/oracle";
+import { cantTell, failed, inapplicable, passed } from "../common/outcome";
+
+test(`evaluate() passes when some input rule passes`, async (t) => {
```

Comment:
And maybe here:
```suggestion
test(`evaluate() passes when R32 passes`, async (t) => {
```
Thought it doesn't make a huge difference, just for consistency